canonical/canonical.com
# Apply nav updates

## Done

- Added fade functionality when nav item is open
- Added /projects to company bubble
- Reinstated products link in nav

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-953.demos.haus/
- See that the requested changes have been made

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-4977
